requirements.txt

```diff
@@ -97,4 +106,3 @@ websocket-client==1.5.0
 widgetsnbextension==4.0.5
 wordcloud==1.8.2.2
 zipp==3.12.0
-black
```

Comment:
black has been added at the top @Ayush1695

Reply:
it wont cause issue